apps/speed.c

```diff
@@ -774,7 +774,7 @@ static int EVP_Update_loop(void *args)
     else
         rc = EVP_EncryptFinal_ex(ctx, buf, &outl);
 
-    if (rc <= 1)
+    if (rc == 0)
```

Comment:
Would it be better to use `!= 1`? Yes, they should be equivalent if the function only returns 0 or 1, but go for "not success" rather than "is failure"

Reply:
I have no preference